apollo-common/src/main/resources/application.yaml

```diff
@@ -0,0 +1,57 @@
+#
+# Copyright 2022 Apollo Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+spring:
```

Comment:
move the config in `application.properties` to `application.yaml`, h2 works again because `application.yaml`'s priority is lower than `application.properties`, and there is no cover each other.

Reply:
It looks good to me now. I think we could also change the deployment documentation to guide the user on how to specify different profiles without rebuilding the whole project?

Reply:
In chinese document, I move

* `2.2.1.2.7` -> `2.2.3.1`
* `2.2.1.2.8` -> `2.2.3.2`
* `2.2.1.2.9` -> `2.2.3.3`
* `2.2.1.2.10` -> `2.2.3.4`
* `2.2.1.2.11` -> `2.2.3.5`

and split to 2 content, i.e

For version 2.1.0 and above, copy the old content and change the build part.

For version 2.0.0 and below, just copy the old content.

Is it ok?